src/openzaak/js/components/admin/zaaktypen/selectielijst.js

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: EUPL-1.2
+// Copyright (C) 2020 Dimpact
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import useAsync from 'react-use/esm/useAsync';
+
+const PROCESTYPEN_ENDPOINT = '/admin/api/v1/catalogi/selectielijst/procestypen';
+
+
+const ProcestypeOptions = ({ year, initialValue='' }) => {
```

Comment:
I noticed locally that this initial value doesn't seem to change when changing the `year`, seems like a bug?

Reply:
as discussed - this was on purpose to "freeze" the current value in case you reset the year again to what was stored